src/app/renderer/components/app/FileBrowser/components/Sidebar/Sidebar.tsx

```diff
@@ -87,4 +113,41 @@ export class Sidebar extends React.Component<{}, {}> {
     private openFolder = () => {
         openFolderExternally();
     };
+
+    private onVariantFilterSelected = (filter: string) => {
```

Comment:
pls rename this to onSelectVariantFilter. So the naming is consistent

Reply:
Done